From 095c01be0f30f0f8535218889cb19e3c2f583714 Mon Sep 17 00:00:00 2001
From: Vignesh Viswanathan <quic_viswanat@quicinc.com>
Date: Wed, 9 Oct 2024 12:31:06 +0530
Subject: [PATCH] remoteproc: qcom: Remove q6v5_userpd_copy_bootargs

This was added as a temporary change during IPQ5424 bring up and not
required now, so remove q6v5_userpd_copy_bootargs function declaration.

Change-Id: I068a5c5d7607fde4968bb65f92acf04ec8d85a03
Signed-off-by: Vignesh Viswanathan <quic_viswanat@quicinc.com>
---
 drivers/remoteproc/qcom_common.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/remoteproc/qcom_common.h b/drivers/remoteproc/qcom_common.h
index 839fe34f94c4..9f3fb11224aa 100644
--- a/drivers/remoteproc/qcom_common.h
+++ b/drivers/remoteproc/qcom_common.h
@@ -77,5 +77,4 @@ static inline bool qcom_sysmon_shutdown_acked(struct qcom_sysmon *sysmon)
 #endif
 s8 qcom_get_pd_asid(struct device_node *node);
 
-int q6v5_userpd_copy_bootargs(struct rproc *rproc, void *data);
 #endif
-- 
2.34.1

